src/mm-grid/mm-grid.js

```diff
@@ -76,15 +76,17 @@
 		},
 
 		setInitialColumnWidth: function() {
-			var setInitialWidth = this.columns.every(function(column){
-				return column.width === null || column.width === undefined;
-			});
-
-			if(setInitialWidth) {
-				var initialWidth = 100 / this.columns.length;
-				this.columns.forEach(function(column) {
-					column.width = initialWidth + "%";
+			if(this.$.columns.getDistributedNodes().length > 0) {
```

Comment:
Instead of doing this we should just move the setInitialColumnWidth call from domReady to this line: https://github.com/MediaMath/strand/pull/10/files#diff-b32575296e7536ce78cdd6bb8ccb6b76L63